egs/librispeech/WSASR/README.md

```diff
@@ -0,0 +1,142 @@
+# Introduction
+
+This is a weakly supervised ASR recipe for the LibriSpeech (clean 100 hours) dataset. We train a
+conformer model using Bypass Temporal Classification (BTC)/Omni-temporal Classification (OTC) with transcripts with synthetic errors. In this README, we will describe
+the task and the BTC/OTC training process.
```

Comment:
Could you add a link to the BTC/OTC paper here for reference?

Reply:
Added.